gobblin-restli/gobblin-flow-config-service/gobblin-flow-config-service-server/src/main/java/org/apache/gobblin/service/FlowConfigsResource.java

```diff
@@ -46,12 +49,17 @@
 import org.apache.gobblin.runtime.api.SpecNotFoundException;
 import org.apache.gobblin.runtime.spec_catalog.FlowCatalog;
 
+import avro.shaded.com.google.common.collect.ImmutableSet;
+
+
 /**
  * Resource for handling flow configuration requests
  */
 @RestLiCollection(name = "flowconfigs", namespace = "org.apache.gobblin.service", keyName = "id")
 public class FlowConfigsResource extends ComplexKeyResourceTemplate<FlowId, EmptyRecord, FlowConfig> {
   private static final Logger LOG = LoggerFactory.getLogger(FlowConfigsResource.class);
+  private static final Set<String> REQUIRED_METADATA = ImmutableSet.of("delete.state.store");
```

Comment:
Maybe name this as ALLOWED_METADATA instead of REQUIRED_METADATA since is not required, but will be propagated if set.

Reply:
done